app/config/settings.py

```diff
@@ -0,0 +1,12 @@
+import os
+from dotenv import load_dotenv, find_dotenv
+load_dotenv(find_dotenv())
+
+user_name = os.environ.get('MONGO_USER')
+password = os.environ.get('MONGO_PASSWORD')
+DB_NAME= os.environ.get('MONGO_DB')
+MONGODB_URI ="mongodb+srv://"+user_name+":"+password+"@ruas-app.63rrs.mongodb.net/"+DB_NAME+"?retryWrites=true&w=majority"
```

Comment:
Take the entire URL from an environment variable, instead of taking username and password.